From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ian Wahbe <ian@wahbe.com>
Date: Thu, 16 Mar 2023 16:54:28 +0100
Subject: [PATCH] fork


diff --git a/go.mod b/go.mod
index d7bcc0f..3ef0924 100644
--- a/go.mod
+++ b/go.mod
@@ -1,4 +1,4 @@
-module github.com/terraform-providers/terraform-provider-ns1
+module github.com/ns1-terraform/terraform-provider-ns1
 
 go 1.24
 
diff --git a/main.go b/main.go
index 97093e2..dd93b8b 100644
--- a/main.go
+++ b/main.go
@@ -6,7 +6,7 @@ import (
 	"log"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
-	"github.com/terraform-providers/terraform-provider-ns1/ns1"
+	"github.com/ns1-terraform/terraform-provider-ns1/ns1"
 )
 
 func main() {
diff --git a/ns1/resource_record.go b/ns1/resource_record.go
index 371d15d..534f19e 100644
--- a/ns1/resource_record.go
+++ b/ns1/resource_record.go
@@ -107,14 +107,7 @@ func recordResource() *schema.Resource {
 				Type:     schema.TypeList,
 				Optional: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
-				Deprecated: `short_answers will be deprecated in a future release.
-It is suggested to migrate to a regular "answers" block. Using Terraform 0.12+, a similar convenience to "short_answers" can be achieved with dynamic blocks:
-  dynamic "answers" {
-    for_each = ["4.4.4.4", "5.5.5.5"]
-    content {
-      answer  = answers.value
-    }
-  }`,
+				Deprecated: `short_answers will be deprecated in a future release. It is suggested to migrate to a regular "answers" block.`,
 			},
 			"answers": {
 				Type:     schema.TypeList,
